sql/sql_parse.cc

```diff
@@ -7964,7 +7950,7 @@ static void wsrep_prepare_for_autocommit_retry(THD* thd,
 
   /* DTRACE begin */
   MYSQL_QUERY_START(rawbuf, thd->thread_id,
-	            (char *)(thd->db.str ? : thd->db.str : "unknown"),
+	            (char *)(thd->db.str ? thd->db.str : "unknown"),
```

Comment:
Any reason not to use `thd->get_db()` here for symmetry with other `MYSQL_QUERY_START()`?